### ISSUE TITLE
image in news listing not wide enough

### ISSUE THREAD

**Issue by dkevey** [unimelb/unimelb-design-system#746]
#### Expected behaviour

Using the 5 col block listing images were cropped at a ratio of 3:2  and now with 4 col display they seem to work better at a ratio of 2:1 however there are legacy news and event items that are now displaying with white space to the right. 
#### Actual behaviour

Images are set to display using **max-height=100%** which outputs white space.

![max-height](https://cloud.githubusercontent.com/assets/1305804/19459276/65418160-951d-11e6-9fe4-f82a6d7c80a5.png)
#### Steps to reproduce

Would be better to display using **max-width=100%** . Images may appear slightly stretched, but if using 3:2 or 2:1 ratio they display ok.
[](http://law.unimelb.edu.au/news/alc)
![image-max-width](https://cloud.githubusercontent.com/assets/1305804/19459300/82847e62-951d-11e6-9ca7-a48d67158135.png)

**Comment by tomstringer** [comment]
Have added a temp fix in MCE:

```
/* Fix image sizes in block-listings https://github.com/marcom-unimelb/unimelb-design-system/issues/746 */
.uomcontent [role="main"] .block-listing .crop-height img { 
    max-height: none; 
    max-width: 100%
}
```

**Comment by waitingallday** [comment]
Oh yeah I forgot that one, we're trying to figure out here why the default behaviour is the way it is. Will likely include this in a future update.

**Comment by axelboc** [comment]
Hmm I don't think this is fixed @waitingallday. Only fix is to use background images. Keeping this one closed in favour of #709, which is the same issue.
